src/segment_regquery_others.go

```diff
@@ -0,0 +1,7 @@
+//go:build !windows
+
+package main
+
+func (r *regquery) enabled() bool {
+	return false
```

Comment:
Rather than split this, I would a simply do the following in this case:

```golang
if r.env.getRuntimeGOOS() != windowsPlatform {
    return false
}
```

We will never implement the registry segment for any other platform.

Reply:
done